### PR TITLE
Added Solutions/Krt-async-stuck on behalf of Suryakant Pandian

### PR DESCRIPTION
**Pull request by ipmonk**

### CODE REVIEW

**Comment by ayushgithub** [comment]
jenkins please retry a build

**Comment by ayushgithub** [comment]
jenkins please retry a build

**Comment by ayushgithub** [comment]
jenkins please retry a build

File: juniper_official/Solutions/Krt-async-stuck/krtasync-rule.conf [hunk 1]
```diff
@@ -0,0 +1,226 @@
+iceberg {
```
Comment (ayushgithub) [inline juniper_official/Solutions/Krt-async-stuck/krtasync-rule.conf]: 461
Entering configuration mode
|/rules/juniper_official/Solutions/Krt-async-stuck/krtasync-rule.conf:223:(19) syntax error: system-settings
/rules/juniper_official/Solutions/Krt-async-stuck/krtasync-rule.conf:225:(5) error recovery ignores input until this point: }
load complete (2 errors)

File: juniper_official/Solutions/Krt-async-stuck/krtasync.playbook [hunk 2]
```diff
@@ -0,0 +1,156 @@
+This playbook is used to detect KRT Async queue stuck condition and raise alarm.
```
Comment (ayushgithub) [inline juniper_official/Solutions/Krt-async-stuck/krtasync.playbook]: 461
Entering configuration mode
The configuration has been changed but not committed
|/rules/juniper_official/Solutions/Krt-async-stuck/krtasync.playbook:1:(4) syntax error: This
/rules/juniper_official/Solutions/Krt-async-stuck/krtasync.playbook:156:(5) error recovery ignores input until this point: }
load complete (2 errors)

File: juniper_official/Solutions/Krt-async-stuck/krtasync.playbook [hunk 3]
```diff
@@ -0,0 +1,157 @@
+/ *
```
Comment (ayushgithub) [inline juniper_official/Solutions/Krt-async-stuck/krtasync.playbook]: 480
Entering configuration mode
The configuration has been changed but not committed
|/rules/juniper_official/Solutions/Krt-async-stuck/krtasync.playbook:1:(1) syntax error: /
/rules/juniper_official/Solutions/Krt-async-stuck/krtasync.playbook:157:(5) error recovery ignores input until this point: }
load complete (2 errors)

File: juniper_official/Solutions/Krt-async-stuck/krtasync.playbook [hunk 4]
```diff
@@ -0,0 +1,157 @@
+/*
```
Comment (ayushgithub) [inline juniper_official/Solutions/Krt-async-stuck/krtasync.playbook]: 481
Entering configuration mode
The configuration has been changed but not committed
|/rules/juniper_official/Solutions/Krt-async-stuck/krtasync.playbook:155:(8) syntax error: playbook
/rules/juniper_official/Solutions/Krt-async-stuck/krtasync.playbook:157:(5) error recovery ignores input until this point: }
load complete (2 errors)

File: juniper_official/Solutions/Krt-async-stuck/krtasync-rule.conf (hunk 1 above)
Comment (ipmonk) [reply juniper_official/Solutions/Krt-async-stuck/krtasync-rule.conf]: seems to be error with CI not code:

Setting status of 0f874b7c1b8a8bcfa098f56d6c4c904ec6c97547 to FAILURE with url http://10.209.18.90:8081/job/iceberg/job/jfit-rules_validate/482/ and message: 'Build finished. '
ERROR: Error fetching remote repo 'origin'
Finished: FAILURE

File: juniper_official/Solutions/Krt-async-stuck/krtasync-rule.conf [hunk 5]
```diff
@@ -0,0 +1,205 @@
...
```
Comment (ayushgithub) [inline juniper_official/Solutions/Krt-async-stuck/krtasync-rule.conf]: 489
Entering configuration mode

error: problem checking file: No such file or directory
load complete (1 errors)

File: juniper_official/Solutions/Krt-async-stuck/krtasync.playbook [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
```
Comment (ayushgithub) [inline juniper_official/Solutions/Krt-async-stuck/krtasync.playbook]: 489
Entering configuration mode

error: problem checking file: No such file or directory
load complete (1 errors)

**Comment by ayushgithub** [comment]
jenkins please retry a build

**Pull request merged**